shared-module/displayio/Display.c

```diff
@@ -299,11 +309,11 @@ void displayio_display_set_region_to_update(displayio_display_obj_t* self, displ
     if (self->colorspace.depth < 8) {
         uint8_t pixels_per_byte = 8 / self->colorspace.depth;
         if (self->colorspace.pixels_in_byte_share_row) {
-            x1 /= pixels_per_byte;
-            x2 /= pixels_per_byte;
+            x1 /= pixels_per_byte * self->colorspace.bytes_per_cell;
```

Comment:
`pixels_per_byte * self->colorspace.bytes_per_cell` could be computed just once. We'd hope the compiler would take care of this.

Reply:
ya, I hope it would too.